packages/core/test/snapshot-setup.ts

```diff
@@ -0,0 +1,31 @@
+import path from 'node:path'
+import { snapshot } from 'node:test'
+
+snapshot.setDefaultSnapshotSerializers([
+	(value) => {
+		if (typeof value === 'string') {
+			return value
+		}
+		return JSON.stringify(value, (_k, v) => typeof v === 'bigint' ? String(v) : v, 2)
+	},
+])
+
+snapshot.setResolveSnapshotPath((testPath) => {
+	if (!testPath) {
+		throw new Error("testPath is undefined. This shouldn't happen as we are not in a REPL")
+	}
+
+	const packagesIndex = testPath.lastIndexOf(`${path.sep}packages${path.sep}`)
+	if (packagesIndex === -1) {
+		throw new Error(`Could not find '/packages/' directory in testPath: ${testPath}`)
+	}
+
+	return path.join(
+		testPath.slice(0, packagesIndex),
+		'__snapshots__',
+		testPath.slice(packagesIndex + 1)
+			// Directory is called /test-out/ for historical reasons
+			.replace(new RegExp(`${path.sep}test${path.sep}`), `${path.sep}test-out${path.sep}`)
```

Comment:
Goal is to keep the new snapshot files at the exact same locations as the old ones, so decided to keep the `test-out` part in the snapshot file paths.

Reply:
Imo it would be acceptable to move them to the default location, now that it's been verified that the snapshots are correct.